editors/document-model/editor-schema.tsx

```diff
@@ -127,16 +104,16 @@ export default function EditorSchema({
         if (!editorRef.current) {
             return;
         }
-
-        editorRef.current.onDidBlurEditorText(() => {
+        const listener = editorRef.current.onDidBlurEditorText(() => {
             const value = editorRef.current?.getValue();
             if (value) {
-                generateSchema(value);
+                generateSchema(value, name);
             } else {
                 // TODO clear current schema?
             }
         });
-    }, [editorRef.current]);
+        return listener.dispose;
+    }, [editorRef.current, name]);
```

Comment:
You probably don't want to have `editorRef.current` in this dependency array

https://epicreact.dev/why-you-shouldnt-put-refs-in-a-dependency-array/